### PR TITLE
[DowngradePhp80] Add Static Call Support on DowngradeMatchToSwitchRector

### PR DESCRIPTION
**Pull request by samsonasik**
Continue of https://github.com/rectorphp/rector-src/pull/2173 by @jrstanley, this PR add static call support as well.

### CODE REVIEW

**Comment by samsonasik** [comment]
implemented 🎉

File: rules/DowngradePhp80/Rector/Expression/DowngradeMatchToSwitchRector.php [hunk 1]
```diff
@@ -156,7 +154,7 @@ private function createSwitchStmts(Echo_ | Expression | Return_ $node, MatchArm
         } elseif ($node instanceof Echo_) {
             $stmts[] = new Echo_([$matchArm->body]);
             $stmts[] = new Break_();
-        } else if ($node->expr instanceof MethodCall || $node->expr instanceof FuncCall) {
+        } elseif ($node->expr instanceof CallLike) {
```
Comment (TomasVotruba) [inline rules/DowngradePhp80/Rector/Expression/DowngradeMatchToSwitchRector.php]: Here should be probably any `Expr`, e.g. `echo` etc.

Comment (samsonasik) [reply rules/DowngradePhp80/Rector/Expression/DowngradeMatchToSwitchRector.php]: `Echo_` is a statement, already handled before, and this check is fill against its `args` property of object, which only exists on class child of `CallLike`, I moved the check in the last elseif

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

File: rules/DowngradePhp80/Rector/Expression/DowngradeMatchToSwitchRector.php (hunk 1 above)
Comment (TomasVotruba) [reply rules/DowngradePhp80/Rector/Expression/DowngradeMatchToSwitchRector.php]: I see :+1:

**Comment by jrstanley** [comment]
Thank you @samsonasik @TomasVotruba for the `match` downgrade love!

**Pull request merged**